src/Illuminate/Broadcasting/Broadcasters/AblyBroadcaster.php

```diff
@@ -139,62 +193,114 @@ public function broadcast(array $channels, $event, array $payload = [])
     }
 
     /**
-     * Build an Ably message object for broadcasting.
+     * @param $channelName string
+     * @param $token string
+     * @param $clientId string
+     * @param $channelCapability string[]
```

Comment:
```suggestion
     * @param  string  $channelName
     * @param  string  $token
     * @param  string  $clientId
     * @param  string[]  $channelCapability
```
First variable type and then the variable name

Reply:
Fixed!